oauth2-authorization-server/src/main/java/org/springframework/security/oauth2/server/authorization/authentication/OAuth2AuthorizationCodeAuthenticationToken.java

```diff
@@ -39,39 +40,43 @@
 	private Authentication clientPrincipal;
 	private String clientId;
 	private String redirectUri;
+	private final Map<String, Object> additionalParameters;
 
 	/**
 	 * Constructs an {@code OAuth2AuthorizationCodeAuthenticationToken} using the provided parameters.
-	 *
```

Comment:
Woops, auto-formatting mishap. Is there an IDEA config somewhere I could use so this doesn't happen again ?

Reply:
I don't have anything special configured in my IDEA. As a general practise, I always review the changes before I commit so that I can confirm no other changes were made inadvertently.